wasm/logic/variable.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import (
+    cast,
+)
+
+from wasm.datatypes import (
+    GlobalInstance,
+    Mutability,
+)
+from wasm.execution import (
+    Configuration,
+)
+from wasm.instructions import (
+    GlobalOp,
+    LocalOp,
+)
+
+logger = logging.getLogger('wasm.logic.variable')
+
+
+def set_local_op(config: Configuration) -> None:
+    instruction = cast(LocalOp, config.instructions.current)
+    logger.debug("%s()", config.instructions.current.opcode.text)
```

Comment:
Update all opcode logging statements (where possible) to use static TEXT to reduce overhead.

Reply:
Saving this for later cleanup performance improvements cleanup (trading tech debt for expediency)